app/(main)/_components/item.tsx

```
@@ -28,7 +33,35 @@ export const Item = ({
+    const handleExpand = (
+        event: React.MouseEvent<HTMLDivElement, MouseEvent>
+    ) => {
+        event.stopPropagation();
+        onExpand?.();
+    }
+
+    const onCreate = (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+        event.stopPropagation();
+        if (!id) return;
+        const promise = create({ title: "untitled", parentDocument: id })
+            .then((documentId) => {
```

ESLint

[error] 'documentId' is defined but never used.

(@typescript-eslint/no-unused-vars)